Resources/views/sckenhancedselection_content_field.html.twig

```diff
@@ -1,7 +1,16 @@
-{% block ngclasslist_field %}
+{% block sckenhancedselection_field %}
+{% set available_options = fieldSettings.options %}
 {% spaceless %}
-    {% for identifier in field.value.identifiers %}
-        {{ identifier }}{% if not loop.last %}, {% endif %}
-    {% endfor %}
+    {% if available_options|length > 0 %}
+        {% for option in available_options %}
+            {% if option.identifier in field.value.identifiers %}
+                {{ option.name }}{% if not loop.last %}{% if fieldSettings.delimiter %}{{ fieldSettings.delimiter }}} {% else %}, {% endif %}{% endif %}
```

Comment:
Also, you should check if delimiter is not empty with `{% if fieldSettings.delimiter is not empty %}`